src/dbt/kipptaf/models/kippadb/intermediate/int_kippadb__enrollment_pivot.sql

```diff
@@ -167,6 +167,7 @@ with
             e.secondary_enrollment_id as hs_enrollment_id,
             e.vocational_enrollment_id as cte_enrollment_id,
             e.graduate_enrollment_id,
+            e.curr_enrollment_id as cur_enrollment_id,
```

Comment:
kind of unnecessary, could you alias in the reporting view if needed?

Reply:
That was just to keep it consistent with the other cur fields. That one happens to be the only double r.

Reply:
oh I see that now, I'll fix it at the source